src/pages/ProfilePage.tsx

```diff
@@ -21,154 +20,136 @@ import PressableWithoutFocus from '@components/Pressable/PressableWithoutFocus';
 
     const displayName = PersonalDetailsUtils.getDisplayNameOrDefault(details);
-    const avatar = lodashGet(details, 'avatar', UserUtils.getDefaultAvatar());
-    const fallbackIcon = lodashGet(details, 'fallbackIcon', '');
-    const originalFileName = lodashGet(details, 'originalFileName', '');
-    const login = lodashGet(details, 'login', '');
-    const timezone = lodashGet(details, 'timezone', {});
+    const avatar = details?.avatar ?? UserUtils.getDefaultAvatar();
+    const fallbackIcon = details?.fallbackIcon ?? '';
+    const originalFileName = details?.originalFileName ?? '';
+    const login = details?.login ?? '';
+    const timezone = details?.timezone;
 
     // If we have a reportID param this means that we
     // arrived here via the ParticipantsPage and should be allowed to navigate back to it
-    const shouldShowLocalTime = !ReportUtils.hasAutomatedExpensifyAccountIDs([accountID]) && !_.isEmpty(timezone);
-    let pronouns = lodashGet(details, 'pronouns', '');
-    if (pronouns && pronouns.startsWith(CONST.PRONOUNS.PREFIX)) {
+    const shouldShowLocalTime = !ReportUtils.hasAutomatedExpensifyAccountIDs([accountID]) && !isEmptyObject(timezone);
+    let pronouns = details?.pronouns ?? '';
+    if (pronouns?.startsWith(CONST.PRONOUNS.PREFIX)) {
         const localeKey = pronouns.replace(CONST.PRONOUNS.PREFIX, '');
-        pronouns = props.translate(`pronouns.${localeKey}`);
+        pronouns = translate(`pronouns.${localeKey}` as TranslationPaths);
     }
 
     const isSMSLogin = Str.isSMSLogin(login);
     const phoneNumber = getPhoneNumber(details);
     const phoneOrEmail = isSMSLogin ? getPhoneNumber(details) : login;
 
-    const isCurrentUser = props.session.accountID === accountID;
-    const hasMinimumDetails = !_.isEmpty(details.avatar);
-    const isLoading = lodashGet(details, 'isLoading', false) || _.isEmpty(details);
+    const isCurrentUser = session?.accountID === accountID;
+    const hasMinimumDetails = !isEmptyObject(details.avatar);
+    const isLoading = details?.isLoading ?? false ?? isEmptyObject(details);
 
     // If the API returns an error for some reason there won't be any details and isLoading will get set to false, so we want to show a blocking screen
     const shouldShowBlockingView = !hasMinimumDetails && !isLoading;
 
-    const statusEmojiCode = lodashGet(details, 'status.emojiCode', '');
-    const statusText = lodashGet(details, 'status.text', '');
+    const statusEmojiCode = details?.status?.emojiCode ?? '';
+    const statusText = details?.status?.text ?? '';
     const hasStatus = !!statusEmojiCode;
     const statusContent = `${statusEmojiCode}  ${statusText}`;
 
-    const navigateBackTo = lodashGet(props.route, 'params.backTo', ROUTES.HOME);
+    const navigateBackTo = (route?.params?.backTo as Route) ?? ROUTES.HOME;
```

Comment:
Try to update `backTo` type instead
```suggestion
    const navigateBackTo = route?.params?.backTo ?? ROUTES.HOME;
```